src/Utils/Emoji.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+const Emoji = props => (
```

Comment:
You could destructure props here:

```suggestion
const Emoji = ({ label, symbol }) => (
```